homeassistant/components/discovergy/config_flow.py

```diff
@@ -92,14 +86,12 @@ async def _validate_and_save(
                     httpx_client=get_async_client(self.hass),
                     authentication=BasicAuth(),
                 ).get_meters()
-
-                result = {"title": user_input[CONF_EMAIL], "data": user_input}
             except discovergyError.HTTPError:
                 errors["base"] = "cannot_connect"
             except discovergyError.InvalidLogin:
                 errors["base"] = "invalid_auth"
-            except Exception:  # pylint: disable=broad-except
-                _LOGGER.exception("Unexpected exception")
+            except Exception as err:  # pylint: disable=broad-except
+                _LOGGER.error("Unexpected error occurred while getting meters: %s", err)
```

Comment:
We normally log unknown exceptions in the config flow including the stack trace. Do you think it's excessive?